src/components/Layout/SymptomsCards/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './SymptomsCards.scss';
+
+const SymptomsCards = (props) => {
```

Comment:
Here you can use destructuring to get the props:
```js
const SymptomsCards = ({ title, description, image }) => {...}
```
And remember to remove the word `prop` where you used inside the component. ⚠️

Reply:
Done!